test/recipes/30-test_evp_data/evppkey_ecc.txt

```diff
@@ -3598,6 +3598,21 @@ Result = DERIVE_SET_PEER_ERROR
 
 Title=prime256v1 curve tests
 
+PrivateKey=ALICE_cf_prime256v1
+-----BEGIN EC PRIVATE KEY-----
+MHcCAQEEIKuC91IrK77OjrAREc/7mCRDnKhjVIiAdjJldBdGv47YoAoGCCqGSM49
+AwEHoUQDQgAE5usNhOHMXYW1W8/E0TRbMhSvK4a0ipu5+xDoKbhz55WBgERHxLWA
+vZugiDpyoshYqVPseGubEPA4ZpGdieYajA==
+-----END EC PRIVATE KEY-----
+
+PublicKey=ALICE_cf_prime256v1_PUB
+-----BEGIN PUBLIC KEY-----
+MFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAE5usNhOHMXYW1W8/E0TRbMhSvK4a0
+ipu5+xDoKbhz55WBgERHxLWAvZugiDpyoshYqVPseGubEPA4ZpGdieYajA==
+-----END PUBLIC KEY-----
+
+PrivPubKeyPair=ALICE_cf_prime256v1:ALICE_cf_prime256v1_PUB
+
```

Comment:
For the priv/pub key pair being really useful we will need some testcases added with it. I.e., we need Derive/PeerKey/SharedSecret triplet added. Also the names seem to be duplicate with the following priv/pub keys which is wrong.

Reply:
whoops yes name collision thanks @t8m!

but also `PrivPubKeyPair` is a test -- it check the consistency of the given key pair.

Reply:
.. and in that public-key-from-private-key computation, is exactly where the bug is triggered. I'll get @Trust-Worthy to add a little more detail, at least to this thread, bc without the context it just looks like random KATs going in

Reply:
Ah, ok. Then please at least rename the keys